<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresenterWidget</class>
 <widget class="QWidget" name="PresenterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PresenterWidget</string>
  </property>
  <widget class="QListWidget" name="questionList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>260</width>
     <height>379</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="idLabel">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="idTextBox"/>
      </item>
      <item>
       <widget class="QLabel" name="questionLabel">
        <property name="text">
         <string>Question</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="questionTextBox"/>
      </item>
      <item>
       <widget class="QLabel" name="answerLabel">
        <property name="text">
         <string>Answer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="answerTextBox">
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="scoreLabel">
        <property name="text">
         <string>Score</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="scoreTextBox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>questionList</zorder>
  <zorder>answerLabel</zorder>
  <zorder>scoreTextBox</zorder>
  <zorder>questionLabel</zorder>
  <zorder>answerTextBox</zorder>
  <zorder>scoreLabel</zorder>
  <zorder>idTextBox</zorder>
  <zorder>idLabel</zorder>
  <zorder>questionTextBox</zorder>
  <zorder>idLabel</zorder>
  <zorder>idLabel</zorder>
  <zorder>idLabel</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
